config/conf.json

```diff
@@ -73,7 +73,7 @@
             "base_label": "R",
             "tools": ["r"],
             "packages": {},
-            "version": "1.0.2",
+            "version": "1.0.3",
```

Comment:
I think other versions in this file need to be changed as well for: `terra-jupyter-gatk`, `terra-jupyter-aou`, `terra-jupyter-bioconductor`

Reply:
oh right, good catch

Reply:
if the previous versions are bad, shall we just remove the bad versions, keep versions the same in this PR?

Reply:
going to merge as is so we have a full history of the changes made to these images